pytorch_pfn_extras/training/_trainer.py

```diff
@@ -24,39 +36,45 @@ def __init__(self, handler, *, evaluator, **kwargs):
         self.val_loader = None
 
     @property
-    def epoch(self):
+    def epoch(self) -> int:
         return self.manager.epoch
 
     @property
-    def epoch_detail(self):
+    def epoch_detail(self) -> float:
         return self.manager.epoch_detail
 
     @property
-    def iteration(self):
+    def iteration(self) -> int:
         return self.manager.iteration
 
     @property
-    def is_before_training(self):
+    def is_before_training(self) -> bool:
         return self.manager.iteration == 0
 
     @property
-    def stop_trigger(self):
+    def stop_trigger(self) -> Trigger:
         return self._stop_trigger
 
     @stop_trigger.setter
-    def stop_trigger(self, trigger):
+    def stop_trigger(self, trigger: Trigger) -> None:
         self._stop_trigger = trigger
 
-    def get_optimizer(self, name):
+    def get_optimizer(self, name: str) -> torch.optim.Optimizer:
         return self.manager.optimizers[name]
 
-    def set_optimizer(self, name, optimizer):
+    def set_optimizer(self, name: str, optimizer: torch.optim.Optimizer) -> None:
         self.manager.optimizers[name] = optimizer
 
-    def is_epoch_last_iter(self, idx):
+    def is_epoch_last_iter(self, idx: int) -> bool:
         return (idx + 1) == (self.manager._iters_per_epoch)
 
-    def _complete_step(self, idx, outs, *, is_deferred=False):
+    def _complete_step(
+            self,
+            idx: int,
+            outs: 'handler_module.Outputs',
```

Comment:
I looked for this definition in:

https://github.com/pfnet/pytorch-pfn-extras/blob/master/pytorch_pfn_extras/handler.py

but it wasn't there. Can you double check please?

Reply:
Oh, I expected mypy to complain via `/test` but not...?

Reply:
yeah, I also expected it, but test came out clean

Reply:
That is because `mypy: ignore-errors` was added to master branch.